game/theater/controlpoint.py

```diff
@@ -391,6 +393,29 @@ def has_factory(self) -> bool:
                 return True
         return False
 
+    @property
+    def has_helipads(self) -> bool:
+        """
+        Returns true if cp has helipads
+        """
+        return len(self.helipads) > 0
+
+    @property
+    def has_free_helipad(self) -> bool:
+        """
+        Returns true if cp has a free helipad
+        """
+        return False in [h.occupied for h in self.helipads]
```

Comment:
```suggestion
        return not all(h.occupied for h in self.helipads)
```

Reply:
Resolved in 9ea1edf9dbc8b21c7204ca6a71eab847ddc63522